addons/rose/addon/components/rose/form/checkbox/group/index.stories.mdx

```diff
@@ -17,38 +17,87 @@ Renders a form checkboxfield group.
   @onChange={{fn this.onChange}}
   as |group|
 >
-  <group.checkbox
-    @label={{group.item.name}}
-    value={{group.item.id}} />
+  <group.checkbox @label={{group.item.name}} value={{group.item.id}} />
 </Rose::Form::Checkbox::Group>
 ```
+
+<Preview>
+  <Story name="Basic" height="10rem">
+    {{
+      template: hbs`
+    <Rose::Form::Checkbox::Group
+      @name={{name}}
+      @inline={{inline}}
+      @disabled={{disabled}}
+      @items={{items}}
+      @selectedItems={{selectedItems}}
+      @onChange={{onChange}}
+      as |group|
+    >
+      <group.checkbox
+        @label={{group.item.name}}
+        value={{group.item.id}} />
+    </Rose::Form::Checkbox::Group>
+  `,
+      context: {
+        inline: boolean('inline', true),
+        disabled: boolean('disabled', false),
+        items: object('items', [
+          { id: 1, name: 'Foo' },
+          { id: 2, name: 'Bar' },
+          { id: 3, name: 'Baz' },
+        ]),
+        selectedItems: object('selectedItems', []),
+        onChange: action('changed'),
+      },
+    }}
+  </Story>
+</Preview>
+
+```htmlbars
+<Rose::Form::Checkbox::Group
+  @name='checkbox-group'
+  @items={{this.items}}
+  @selectedItems={{this.selectedItems}}
+  @onChange={{fn this.onChange}}
+  @itemEqualityFunc={{this.isEqual}}
```

Comment:
This isn't necessary, right?

Reply:
What do you mean by necessary? This is the example documentation for the new story that includes the `@itemEqualityFunc`. The old one is still [above it](https://github.com/hashicorp/boundary-ui/blob/c90d1e933d942b5a44a12934e0a4dc39329580f0/addons/rose/addon/components/rose/form/checkbox/group/index.stories.mdx?plain=1#L12)